MonoGame.Framework.Content.Pipeline/Graphics/GraphicsUtil.cs

```diff
@@ -159,7 +167,9 @@ public static void CompressDxt(GraphicsProfile profile, TextureContent content,
 
             if (alphaRange == AlphaRange.Opaque)
                 Compress(typeof(Dxt1BitmapContent), content, generateMipMaps);
-            else if (alphaRange == AlphaRange.Cutout || sharpAlpha)
+            else if (sharpAlpha)
+                CompressFontDXT3(content, generateMipMaps);
```

Comment:
`sharpAlpha` was added for font texture compression, and is only used by font processing, so that is the trigger for this special behaviour.  Perhaps renaming the parameter would be better.